src/base/serial/tty_io.c

```diff
@@ -351,10 +352,9 @@ static int tty_lock(const char *path, int mode)
   else if (mode == 2) { /* re-acquire a lock after a fork() */
     FILE *fd;
 
-     fd = fopen(saved_path,"we");
-     if (fd == (FILE *)0) {
-      error("tty_lock: reacquire (%s): %s\n",
-              saved_path, strerror(errno));
+    fd = fopen(saved_path,"we");
+    if (fd == (FILE *)0) {
```

Comment:
NULL? :)

Reply:
I did have `if (!fd)` in a draft commit, but I decided not to change stuff other than what coverity was actually moaning about.